packages/iris-grid/src/AdvancedFilterCreatorSelectValueList.tsx

```diff
@@ -234,7 +270,9 @@ class AdvancedFilterCreatorSelectValueList extends PureComponent {
           onSelect={this.handleSelect}
           onViewportChange={this.handleViewportChange}
           ref={list => {
-            this.list = list;
+            if (list) {
```

Comment:
I don't think you need to wrap this in `if (list)`, we can update the ref to `null` if it is `null`